selfdrive/car/honda/values.py

```diff
@@ -651,6 +673,7 @@ class CAR:
   CAR.PILOT: dbc_dict('honda_pilot_touring_2017_can_generated', 'acura_ilx_2016_nidec'),
   CAR.PILOT_2019: dbc_dict('honda_pilot_touring_2017_can_generated', 'acura_ilx_2016_nidec'),
   CAR.RIDGELINE: dbc_dict('honda_ridgeline_black_edition_2017_can_generated', 'acura_ilx_2016_nidec'),
+  CAR.INSIGHT: dbc_dict('honda_insight_ex_2019_can_generated.dbc', None),
```

Comment:
Don't add .dbc exension. This is why the test hangs.

Reply:
Ah.. Happy little accident, my mistake.